catalog/OpenET/OpenET_GEESEBAL_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,164 @@
+local id = 'OpenET/GEESEBAL/CONUS/GRIDMET/MONTHLY/v2_0';
+local subdir = 'OpenET';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
```

Comment:
Add: `local units = import 'units.libsonnet';`

This brings in the table of units that allows things like `units.millimeter`

https://github.com/google/earthengine-catalog/blob/main/catalog/units.libsonnet